felix.go

```diff
@@ -466,6 +466,12 @@ func servePrometheusMetrics(configParams *config.Config) {
 				prometheus.Unregister(prometheus.NewProcessCollector(os.Getpid(), ""))
 			}
 		}
+		gaugeHost := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "felix_host",
+			Help: "Felix host.",
```

Comment:
For the help, how about "Configured Felix hostname (as a label), typically used in grouping/aggregating stats; the label defaults to the hostname of the host but can be overridden by configuration.  The value of the gauge is always set to 1."